packages/office-ui-fabric-react/src/components/Nav/Nav.scss

```diff
@@ -96,6 +96,11 @@ $linkRightPadding: 20px;
   vertical-align: middle;
 }
 
+.ms-Nav-linkHorizontal {
+  @include float(left);
+  display: inline;
```

Comment:
We shouldn't. Personally not a fan of float left if you can avoid it; display: inline-block is a bit more predictable.